dockers/piped-base-okd/install-nss-wrapper.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The PipeCD Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+BASE_URL="https://ftp.samba.org/pub/cwrap/nss_wrapper"
+VERSION="1.1.11"
+
+echo "Installing nss_wrapper-${VERSION}..."
+curl -L ${BASE_URL}-${VERSION}.tar.gz | tar xvz
+
+cd nss_wrapper-${VERSION}
+mkdir build
+cd build
+cmake -D CMAKE_INSTALL_PREFIX=/usr/local -D CMAKE_BUILD_TYPE=Release ..
+make install
+cd ../..
```

Comment:
This will be wrong if this script is called from another place, so we should avoid something like this.
We can make a temporary directory in `/tmp` and configure `curl` to download `nss_wrapper` to that directory.
After all we can delete that directory.

Reply:
Ah sorry, this is for reverting 2 previous `cd` commands. Never mind.

Reply:
I'm guessing it does remove correctly no matter where it is, because it downloads nss_wrapper the initial working directory, then it moves two times within this script.

Reply:
it is. but thank you for careful consideration.